### ISSUE TITLE
HasDropDown: don't set aria-expanded

### ISSUE THREAD

**Issue by wkeese** [ibm-js/delite#445]
Apparently it's inappropriate to set `aria-expanded` for nodes with popups.   http://heydonworks.com/practical_aria_examples/#submenus doesn't do it, and according to IBM's RPT tool:

> The WAI-ARIA property aria-expanded is not supported global property on element BUTTON that does not have a WAI-ARIA role.

The counterargument is that it's nice for the user to know if the dropdown (aka popup) is shown or not.    But as soon as the popup is shown, focus will shift to the popup and thus aria attributes on the HasDropDown node are meaningless.
